File: app/src/main/java/ru/visualmath/android/api/model/UserInfo.java
package ru.visualmath.android.api.model;

import com.google.gson.annotations.SerializedName;

import java.util.Map;

public class UserInfo {

    @SerializedName("blocks")
    private Map<String, Block> blocks;

    @SerializedName("questionAnswers")
    private Map<String, QuestionAnswer> questionAnswers;

    public Map<String, Block> getBlocks() {
        return blocks;
    }

    public Map<String, QuestionAnswer> getQuestionAnswers() {
        return questionAnswers;
    }
}

File: app/src/main/java/ru/visualmath/android/api/model/UserInfo.kt
package ru.visualmath.android.api.model

import com.google.gson.annotations.SerializedName

data class UserInfo(

        @SerializedName("blocks")
        val blocks: Map<String, Block> = emptyMap(),

        @SerializedName("questionAnswers")
        val questionAnswers: Map<String, QuestionAnswer> = emptyMap()
)
